ee/clickhouse/models/property.py

```diff
@@ -534,26 +534,36 @@ def box_value(value: Any, remove_spaces=False) -> List[Any]:
 
 
 def get_property_values_for_key(key: str, team: Team, value: Optional[str] = None):
```

Comment:
Can we move this function somewhere less generic? e.g. `ee/clickhouse/queries`?

Reason: This file is probably the most important file for querying anything regarding clickhouse, the most generic and important functions live here. This function is used exactly once, making the code harder to follow.